bundle/src/main/java/com/cognifide/cq/cqsm/core/history/HistoryEntryPropsFactory.java

```diff
@@ -0,0 +1,72 @@
+/*-
+ * ========================LICENSE_START=================================
+ * AEM Permission Management
+ * %%
+ * Copyright (C) 2018 Cognifide Limited
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+package com.cognifide.cq.cqsm.core.history;
+
+import static com.cognifide.cq.cqsm.api.history.Entry.SCRIPT_HISTORY_FILE_NAME;
+
+import com.cognifide.cq.cqsm.api.scripts.Script;
+import com.cognifide.cq.cqsm.core.scripts.ScriptImpl;
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.Optional;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.factory.MissingElementsException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HistoryEntryPropsFactory {
```

Comment:
Name of this class suggest that it realizes factory pattern, but it does not. Remove factory suffix from name.

Reply:
Fixed